pom.xml

```diff
@@ -64,6 +64,7 @@
     </scm>
     <properties>
         <janusgraph.compatible.versions />
```

Comment:
Current JanusGraph version is set on graph initialization here https://github.com/rafatuita/janusgraph/blob/3101295393816df4c178678c2b29771954f6e3b3/janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java#L1367